requirements.txt

```diff
@@ -0,0 +1,2 @@
+aiohttp==2.2.3
```

Comment:
`requirements.txt` is more generally used in Python projects when you look for the requirements. I expect most Python developers to expect either a `requirements.txt` or no `requirements.txt` at all because the dependencies are in the `setup.py`.

My true goal here is only to get one place to maintain the dependencies. I can move it to `requirements/base.txt` or something if you prefer?